### PR TITLE
Fix issue workchain viewer not designated for workchains except QeAppWorkchain

### LINKED ISSUE #344: Bug report: node view check the process label too strigent
## Additional comments (optional):

_Example: I submitted a band structure calculation._

Expend the QeAppWorkChain details and cilck the PwRelaxWorkChain, the exception raised.

## Attachments

<details>
<summary>Traceback</summary>

```python-traceback
KeyError                                  Traceback (most recent call last)
~/apps/quantum-espresso/aiidalab_qe/widgets.py in _observe_node(self, change)
    382                 clear_output()
    383                 if change["new"]:
--> 384                     display(viewer(change["new"]))
    385 
    386 

~/.local/lib/python3.9/site-packages/aiidalab_widgets_base/viewers.py in viewer(obj, downloadable, **kwargs)
     82         raise
     83     else:
---> 84         return _viewer(obj, downloadable=downloadable, **kwargs)
     85 
     86 

~/apps/quantum-espresso/aiidalab_qe/node_view.py in __init__(self, node, **kwargs)
    508     def __init__(self, node, **kwargs):
    509         if node.process_label != "QeAppWorkChain":
--> 510             raise KeyError(str(node.node_type))
    511 
    512         self.node = node

KeyError: 'process.workflow.workchain.WorkChainNode.'
```
</details>

<details>
<summary>Environment fingerprint</summary>
<pre>eNptWF1v47gV_SuB-7oSLCl24nmbDtCHxUybNrMdFEURUBItcyyRHJJKoizmv_d-0VbaffQhRd6Pc8-99O-bZx2icXbzofpl40eV
ji5Mmw-_b1ToTibpLs1Bbz78e7O_bU3a_LLZ_Af2Lenk7NPl001THsqqgdUr9Kd6e1v81s42zTePXx5u_hLMzcd5uKn3N9XhQ739
UFc3v339dFNv63rzE85U3VkNOtLdxvSq6BzevKnLuqzgaAZ91HPvAN6W9-X2Av-YFVw06eiDjhHXb8vquj6qFk-
Cg96DhR4tnVWV9Ro-uUn_8Qc_CG_K_4OLFxfO3UkZ-4cbXkw_6BSLVkV2altuW3bLFd6cFYB7cEmgMP2gsDYMjHpqTQdIlbfYaOp
TmkbB6C67GEdf7XmP970JkXbclreMTK7X6-hhlp914VU6EXzLgQgD_6z3-bezddEdjwaNr8Cu7Xu4aI3tjR0ir9e8Tr42ZS0ZjCm
5s7ZRAnBP2GK7YtBWB5VcIGMr-jYlsh2_pcP-rFo90pfVtkSutcCYTo0jG0p7EPIupFgeZ9sl58ZYjGEuOtWdNJ29p0C0XVh8Ytv
IwVarOZnjPEY3-1umT0VGt6OGjwHZgcUMGHvWbOkOf4NLJ7Lrls0KLo3GL2TWHZn1Ca__5GwKjq2tkFiwQHaRmXRlTcZ1K9fZrU4
HMI5Cv8V41OVdRuWeiq6W_IDhOzIV6BiiToUFPqnRvJHVGE9e1JOxRycnHPgmE7o58pHMkG403RmA-3wH_obatEczFEczMpv2171
F9MZaugrtomR2bnRBTUootiPI9oqdlKsRKEQGipOyPQR6IHcuG-BO136XZNwzltwcyAegCccFc-uGoPwJ4eaAtYb40o3ZLKZUrzq
QOOEFA0l1o4pRcwzwil638yAX7CkEvQZpErbuJJq9Ps5R96_TKHlnEOQI0smKmIncm3guwA3Q1phY43IAextZW2k7F1k_jy-mk3J
kndKgB9omKdq63JFZelJmLJ4h0f2llFg_9DQBC7p8V3NPPIMTIFLO2MTOEvbaaY_2DgEKgblEVuhX3UGFSELY5yMUtPKGI3W4QN-
jsxHZpdhplhCkyuiISo0k9Dia9CRWNRKyIyT0WhANc_A4cxdiVqKvlN1n88ZR4cIcgtZ21EksxG2neVL2GIwGMlEGmQmmt4ruRJf
NhIoxmraAIClkQC7_96vQWIBpHTFjB4WJ50P5gRRY0qU9erplcPKjFFXNgOT0rmyavAchFD6I6hhXGoZL4Ikg8r20Dzri7n_AQr8
mcFD3q9SYFHtlBw1ZjEI9_Oi77k0OI2771djvqqZQyAYH8tbSQWwNJnMnvuzkN3FGtKQRLOgjWyy_LwRo8kEz-
KXDE2gEEI9jwSfmBSbCRXgFL6IOzyK31eHdSqI6ABKL1KyuOc0t3b1dgzwFQNaprq9g4ZcB6ymuYn5dfboYUKOuvl8sronJoT-bl
6v63wkQ3Zid4FY8qnAGsQN5DrkjbAlOUILQozvpkXgiZGRSU6tEhNhRrKWL_FZs9OiGOcwrPfmizk6SiYmiU1ZnfAEjevdiJSqVQ
LN_VEe9MmBSyY_Y1VoZL5p3YGEsSLVYUu5xzcQ023wE-jtBXS-5gR3KA0FBF6C_Idc7cPKW7Jp8kXWl2RJJJj-x2FUyDU5xwFYhs
k83zGMyvekS1SJjVqUI5Utnc4BtS_pOw1RuRYgJKXPHBshZyBgTla-0Lc7GKlH984hldUwFjTCG47wj961OOA2-MgURGMZno1-Ek
hg965JunTuTsRD6eoUV8WSmFaftPEnqa06Sg2HllLPBxjlQ7Un1et26M1Zc3JPoVxh-7rSs6DXLpYdBMyXVjlqGzy2Bkx6VBFqAH
37VoDy0ahXF_UYA1wE5IbcZ540wAxhiZJ0JjyWQx3n-Np0yH_JY7vWr15TXWxlb_RBnKVb6CMaOUUeYdrTkENP6YMbRYcwPJdPqw
UiOK86eN54HECaLPw-oxU8o9FCtT1ZNOjdRuhTZvnDPF89HyQx1ZrIkwMshnfQcixWnKk4bLvpUoHtnkzIZaCXizXQ0twwfl875o
aZQHfhogXDSVqR5l5W0wNHQmeKKwR46ZqFhGliJml9akBcGJCTLJ3P8xzcKLReDh1uiSiIiDSMXoYJzKobmMFJt3PKo6RfoN0neK
BX3gYerrNY5ZA_Lr9--ytBCZi5QUdAFr-SkCx6WL8vj3z9L42kI-av6NK7ZtPzNa_v4-JnfW8LNBU1lUiNGRF8AiemSDG59D8sjK
CjT805coJYME4CWbNRAt_q6gn2tAI0dJBTb_Jn73qnJryoWW7hiYvCQAcAbu1iJN__6-OUzKxUtv9GDD7NCh8J7MldBLa4F_WMGu
WGoZicydnngYLTu1wuTE5GU0Sccu-Zwfye8pq0zFni5KBldqztOwQovOxYb5NH-
slTgUpHn-C02x3u8InYG6F3AuynYzLWa8dzB2Jao9dlnWT9Qr3yEQab-
GlQ8rR628ISZfQ7HfiedKp5A2efZ9CJN9KCK5pVv4LhGa-BJ5MRbQmCsjUY_axGwuvQuJvrUD3n0ga-xFICAH0ecY5bcuGkfPBHO
hcyJ0J1KAQNqp4xtrG0xBWhHBYa8aQ5cmwgvWc5l0gHFnbHxiwyS2ABd8XGyyMuLoQAPNiX_foiH6RS06j0Ep0ujDHKEG7t0Mdar
wSS5nGEuT_xtVkTGCL-thxF4RMh1PMKnH_1ESrHnDADHTRp5-tmJxoIUQfU90UiK_wRFkRZcm7GAaC9XMDzRocMqUGVDUzXajgGa
_RHYX8juPEWD4FwfspKQhnHIW8P9cc80-KeCqX1ZtZAXlbpT74bsP9bkSwfDm_ztUVPCX3RLL1UZ52qGtIWBTP7ayPkGOEJl4asq
y3wlvPymw_lNz8Nq7H45ac2hb1hqeGosWmiUEduKnLy9LD15HYzDaebSjavVcrTtJcKrf3xe4FUkzx-2BZ6jl0Gc6IYBW1TIfxuh
cW_G-8uj7OfP_wKgxqep</pre>
</details>

**By submitting this issue I confirm that I am aware that this information can
potentially be used to determine what kind of calculation was performed at the
time of error.**

### PR DESCRIPTION
**Pull request by unkcpz**
~~Combined with https://github.com/aiidalab/aiidalab-widgets-base/pull/430 should fix #344~~

fixes #344 

The `register_viewer_widget` decorator registers the viewer by class and the new viewer for QeAppWorkchain specifically also become the viewer of other WorkChain. It will raise the `KeyError` and raised which is not expected. 
Instead of raising the exception, nothing is returned. This is a workaround since ideally if there is no new specific viewer defined for a specific work chain, it needs to fall back to the native one of AWB. As tried in https://github.com/aiidalab/aiidalab-widgets-base/pull/430

### CODE REVIEW

File: aiidalab_qe/node_view.py [hunk 1]
```diff
@@ -564,12 +565,6 @@ def toggle_camera():
             children=[self.title, self.result_tabs],
             **kwargs,
         )
-        self._process_monitor = ProcessMonitor(
```
Comment (danielhollas) [inline aiidalab_qe/node_view.py:567]: Why was this removed? This seems important to keep the viewer up-to-date with a running process.

**Review by danielhollas — COMMENTED** [review]
I have one question, otherwise looks good.
(though I haven't tested it since I don't work with QeApp generally)

File: aiidalab_qe/node_view.py (hunk 1 above)
Comment (unkcpz) [reply aiidalab_qe/node_view.py:567]: I think you are correct, I removed it since I thought it was a leftover code where I don't understand why it appears here. But as you said, it will update for the running process, I'll give it a test and add the comment above this code.

**Comment by unkcpz** [comment]
> (though I haven't tested it since I don't work with QeApp generally)

Don't worry 😄, I'll ask @superstar54 to give it a test after I revert the wrong change. Thanks a lot for reviewing.

File: aiidalab_qe/node_view.py (hunk 1 above)
Comment (unkcpz) [reply aiidalab_qe/node_view.py]: reverted.

**Comment by unkcpz** [comment]
@superstar54, can you give a look at this? Thanks!

**Comment by superstar54** [comment]
I test the example reported in https://github.com/aiidalab/aiidalab-qe/issues/344. No error anymore. :+1:

**Comment by unkcpz** [comment]
Thanks @superstar54. Can you approve it then? :smile:

**Review by superstar54 — APPROVED** [review]
LGTM!
